internal/server/models/types.go

```diff
@@ -37,6 +37,9 @@ func (s CommaSeparatedStrings) Value() (driver.Value, error) {
 }
 
 func (s *CommaSeparatedStrings) Scan(v interface{}) error {
+	if v == nil {
+		return nil
+	}
```

Comment:
Calling this out. This started failing because I'm using `db.Raw` in the migration queries instead of gorm. This allows us to accept null values from fields for this type.